### ISSUE TITLE
Feature request : Add chunk limit time

### ISSUE THREAD

**Issue by Webonaute** [smbarbour/ChunkyBlocks#1]
unactive chunk after X time the bloc have been placed

configured by group permission so we can have normal user to be able to put 1 bloc that can stay 24 hour, and for VIP group, they can put a chunk loader valid for 1 week for exemple.

**Comment by StoneLegion** [comment]
I would love this. For me I don't care much about the time per group level but I would love to be able to say it lasts 24 hours then gets removed. This would help a lot of lag. Maybe even a cool message letting the user know that this would be undone in 24 hours.

**Comment by Awesomebilly** [comment]
Good idea, I haven't installed this yet, but this would work for large servers with 24 hour removal.
